examples/dswx_s1_sample_runconfig-v3.0.0-er.1.0.yaml

```diff
@@ -0,0 +1,200 @@
+
+                # The Error Code base value added to the offset values to make
+                # error codes unique per-PGE
+                ErrorCodeBase: 100000
+
+                # Path to the Yamale schema used to validate the SAS portion
+                # of the RunConfig
+                # Path should correspond to the file system within the Docker
+                # container, and typically should reference a schema file bundled
+                # with the opera_pge installation directory within the container
+                # Consult the Docker image build scripts for more info
+                SchemaPath: /home/conda/opera/pge/dswx_s1/schema/dswx_s1_sas_schema.yaml
+
+                # Path to the Jinja2 template used to generate the ISO xml
+                # metadata file
+                # Path should correspond to the file system within the Docker
+                # container, and typically should reference a template file bundled
+                # with the opera_pge installation directory within the container
+                # Consult the Docker image build scripts for more info
+                IsoTemplatePath: /home/conda/opera/pge/dswx_s1/templates/OPERA_ISO_metadata_L3_DSWx_S1_template.xml.jinja2
+
+            QAExecutable:
+                # Set to True to enable execution of an additional "Quality Assurance"
+                # application after SAS execution has completed
+                Enabled: False
+
+                # Path to the executable to run, path must be reachable from
+                # within the Docker container (i.e. findable with a "which" command)
+                ProgramPath:
+
+                # List of command-line options to use with ProgramPath
+                ProgramOptions:
+
+            DebugLevelGroup:
+                # Set to True to enable Debug mode (TODO this is currently a no-op)
+                DebugSwitch: False
+
+                # Set to True to have the PGE invoke the SAS/QA executables via
+                # a shell, rather than a Python subprocess
+                # This allows shell-style syntax to be used in ProgramPath and
+                # ProgramOptions, which can be useful for testing
+                ExecuteViaShell: False
+
+        # SAS-specific RunConfig section
+        # Prior to SAS execution by the PGE, the section below starting at "runconfig"
+        # is isolated into its own YAML file for use with the SAS
+        SAS:
+          runconfig:
+            name: dswx_s1_workflow_default
+
+            groups:
+              pge_name_group:
+                pge_name: DSWX_S1_PGE
+
+              input_file_group:
+                # REQUIRED - Single S1 RTC geotiff or
+                #            multiple burst RTC directory containing
+                #            hdf5 metadata and rtc geotiff (directory or files)
+                #            for open water
+                input_file_path: [
+                  input_data/rtc_data/t047_100908_iw3,
```

Comment:
For the paths in the SAS section, let's be consistent about using absolute paths, for example `/home/conda/input_data/rtc_data/...`